src/components/DuosHeader.js

```diff
@@ -216,7 +216,15 @@ class DuosHeader extends Component {
         'Reviewed Cases Record': {
           link: '/reviewed_cases'
         }
+      },
+      chair: {
+      'Manage DARs': {
+        link: this.state.dacChairPath
+      },
+      'Manage DACs': {
+        link: '/manage_dac/chair'
       }
+    }
```

Comment:
nit: indentation

Reply:
Thinking out loud here: we're storing `dacChairPath` in state and it really should be a prop as it should never change. State is meant to hold things that are variable, and the path really is not. This should instead be a constant that is set in a `useEffect` block, but that's beyond the scope of this PR.